data/src/main/java/me/androidbox/data/local/dao/AttendeeDao.kt

```diff
@@ -0,0 +1,21 @@
+package me.androidbox.data.local.dao
+
+import androidx.room.*
+import kotlinx.coroutines.flow.Flow
+import me.androidbox.data.local.DatabaseConstant
+import me.androidbox.data.remote.model.AttendeeModel
+
+@Dao
+interface AttendeeDao {
+    @Query("SELECT * FROM ${DatabaseConstant.ATTENDEE_TABLE}")
+    fun getAttendee(): Flow<List<AttendeeModel>>
+
+    @Insert(onConflict = OnConflictStrategy.IGNORE)
+    fun insertAttendee(attendeeModel: AttendeeModel)
```

Comment:
Any reason for the `IGNORE` strategy? Wouldn't you rather want to replace them if they exist? That way you also wouldn't need the `@Update` function

Reply:
@philipplackner 
Just wondering, if we use replace instead of ignore. Would that make using update as redundant?

So every time we update we would call the insert instead. Just wondering would that cause any confusion because update has the intention to update.

I guess if it removes some boiler plate code this should be better to use insert.

Reply:
I always use `OnConflictStrategy.REPLACE` since yes, it does remove the need of an `@Update` function. With that strategy, if you insert an event with an ID that already exists in the DB it will replace the existing event (and therefore update it). 
There's also a new `@Upsert` annotation for Room which does exactly that without needing to specify an OnConflictStrategy.